indico/modules/events/registration/templates/management/registration_tag_edit.html

```diff
@@ -0,0 +1,19 @@
+{% from 'forms/_form.html' import form_header, form_footer, form_rows %}
+{% from 'message_box.html' import message_box %}
+
+<div class="registration-tags">
+    {{ form_header(form, id='registration-tag-form') }}
```

Comment:
```suggestion
    {{ form_header(form) }}
```

unused id as well

Reply:
But it looks like we can get rid fo this whole template and replace it with `jsonify_form` in the controller - this is what we do for pretty much all other simple cases where we just render a wtform in a modal dialog...